bluemira/codes/plasmod/api.py

```diff
@@ -336,20 +336,23 @@ class Setup(interface.Setup):
 
     """
 
-    def __init__(self, parent, *args, problem_settings=None, **kwargs):
+    def __init__(self, parent, *args, **kwargs):
         super().__init__(parent, *args, **kwargs)
 
-        self._problem_settings = problem_settings if problem_settings is not None else {}
         self.input_file = "plasmod_input.dat"
         self.output_file = "plasmod_outputs.dat"
         self.profiles_file = "plasmod_profiles.dat"
-        self.io_manager = Inputs({**self.get_new_inputs(), **self._problem_settings})
+        self.io_manager = Inputs(
+            {**self._get_new_inputs(), **self.parent.problem_settings}
```

Comment:
Wasn't this changed to a public method in #748?

Reply:
Oops yeah bad rebase